src/components/popover-container/popover-container.spec.tsx

```diff
@@ -314,6 +315,22 @@ describe("PopoverContainer", () => {
       expect(wrapper.find(PopoverContainerContentStyle).exists()).toBe(true);
     });
 
+    it("should close popover if escape key is pressed", () => {
+      wrapper = render();
+
+      act(() => {
+        wrapper.find(PopoverContainerOpenIcon).props().onClick();
+      });
+
+      wrapper.update();
+      expect(wrapper.find(PopoverContainerOpenIcon).props().tabIndex).toBe(-1);
+      expect(wrapper.find(PopoverContainerContentStyle).exists()).toBe(true);
+
+      act(() => {
```

Comment:
**comment**: is this test complete? We seem to be opening the container and checking that worked (above) - probably not an assertion we need in this test as it's tested elsewhere but it doesn't hurt. But now (in this `act`) we're pressing escape, but there are no assertions afterwards. Presumably we should check the container has actually closed?

Reply:
Yeah, my bad. It should be checking the `PopoverContainer` is closed. I'll get that added in.